db/migrations/01_create_database.sql

```diff
@@ -0,0 +1 @@
+CREATE DATABASE chitter;
```

Comment:
Great use of the migration documents. Very easy to know what tables to create and how to format them